.github/mergify.yml

```diff
@@ -0,0 +1,83 @@
+---
+defaults:
+  actions:
+    queue:
+      name: default
+      method: rebase
+      update_method: rebase
+
+
+# each test should be listed separately, do not use regular expressions:
+# https://docs.mergify.io/conditions.html#validating-all-status-check
+# TODO: Use mergify's recently added 'shared configuration support'
+# to dedup some of the check-x=y repetition in the future.
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=check
+      - check-success=build-server
+      - check-success=build-ad-server
+      - check-success=build-nightly-server
+      - check-success=build-nightly-adserver
+      - check-success=build-client
+      - check-success=build-toolbox
+      - check-success=test-ad--server-kubernetes
```

Comment:
This double dash doesn't look right to me. But I'm being lazy and haven't double checked.

Reply:
> This double dash doesn't look right to me. But I'm being lazy and haven't double checked.

fixed / removed